Assets/Scripts/Core/PathNode.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using UnityEngine;
+using System.Collections;
+
+namespace OperationBlackwell.Core {
+
+	public class PathNode {
+
+		public event EventHandler OnWalkableChanged;
+
+		public int xPos;
+		public int yPos;
+		public PathNode parent;
+		public PathNode north;
+		public PathNode south;
+		public PathNode west;
+		public PathNode east;
+		public bool moveNorth;

```

Comment:
Suggestion: Not sure if we/you want to do this, but can't you add "moveNorth", "moveSouth", "moveWest" and "moveEast" to the same line and declare them all at once, the same for the 4 cardinal directions above. This saves lines, but I am not sure if this improves the readability of the code.